lib/reply.js

```diff
@@ -151,7 +151,11 @@ Reply.prototype.code = function (code) {
 Reply.prototype.status = Reply.prototype.code
 
 Reply.prototype.serialize = function (payload) {
-  return serialize(this.context, payload, this.res.statusCode)
+  if (this._serializer) {
```

Comment:
I would use a null check, it's more performant and also more correct in this case.
```suggestion
  if (this._serializer !== null) {
```

Reply:
@delvedor can you commit as suggested change?

Reply:
I've added you as assignee of this pr, you should be able to commit this now :)

Reply:
`You do not have permission to push to this repository.` :(

Reply:
GitHub is having downtime issues, let's try again in an hour :)
https://status.github.com/messages